tools/AzureRM/AzureRM.psd1

```diff
@@ -64,6 +64,7 @@ RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; RequiredVersion = '3.3.1';
                @{ModuleName = 'AzureRM.CognitiveServices'; RequiredVersion = '0.8.4'; }, 
                @{ModuleName = 'AzureRM.Compute'; RequiredVersion = '3.3.1'; }, 
                @{ModuleName = 'AzureRM.Consumption'; RequiredVersion = '0.2.4'; }, 
+               @{ModuleName = 'AzureRM.ContainerInstance'; RequiredVersion = '1.0.0'; },
```

Comment:
@yolo3301 please change the version of this module to 0.0.1. This is the version we use throughout all new modules. This change also applies to `AzureRM.ContainerInstance.psd1`.

Reply:
Changed to 0.0.1